HospitalWebApp/Controllers/DocSearchController.cs

```diff
@@ -19,6 +19,17 @@ public class DocSearchController : ControllerBase
             _medicationPrescriptionService = medicationPrescriptionService;
             _examinationService = examinationService;
         }
+        
+                
+        [HttpGet]
+        [Route("prescription")]
+        public IActionResult GetAllPrescriptions()
+            => Ok(_medicationPrescriptionService.GetAll());
+
+        [HttpGet]
+        [Route("examination")]
+        public IActionResult GetAllExaminations()
+            => Ok(_examinationService.GetByPatientId(1));
```

Comment:
Why patient id 1 here?

Reply:
This and many other hard-coded ids will be fixed in the current sprint. The reason is, as already mentioned before, we still don't have patient logging. This patient id will be fetched from session/request after we have authorization implemented.

Reply:
You can maybe add todo comment for fixing hardcoded ids. I thought that this method will return all existing examinations because you already have IActionResult GetPatientExaminations(int patientId) method.